rules/do-you-measure-success-using-lean-agile-metrics/rule.md

```diff
@@ -1,39 +1,50 @@
 ---
 type: rule
 title: Do you measure success using Lean-Agile metrics?
-uri: do-you-measure-success-using-lean-agile-metrics
+uri: measure-success-using-lean-agile-metrics
```

Comment:
need to update folder name to match uri

Reply:
Done ✅ - PR for folder changes: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8357